routeros/resource_interface_eoip.go

```diff
@@ -9,7 +9,7 @@ import (
 func ResourceInterfaceEoip() *schema.Resource {
 	resSchema := map[string]*schema.Schema{
 		MetaResourcePath: PropResourcePath("/interface/eoip"),
-		MetaId:           PropId(Name),
+		MetaId:           PropId(Id),
```

Comment:
@vaerh I tested this particular change. Functionally, that works as expected, but it breaks the existing state. The `id` property is outdated in the state, and hence terraform loses the link. That can be easily fixed by reimporting the resource, which will update the `id` property only.

That's kind of a breaking change, but we just released this feature, so I am not sure if that's critical.

Reply:
Ok, let's make it pretty. I have added state migration, please test it. Maybe there are some other requests.